api/v1alpha4/virtualmachine_types.go

```diff
@@ -884,9 +884,19 @@ type VirtualMachineStatus struct {
 
 	// +optional
 
-	// Host describes the hostname or IP address of the infrastructure host
-	// where the VM is executed.
-	Host string `json:"host,omitempty"`
+	// NodeName describes the observed name or IP address of the node where the
```

Comment:
It is the host's object name. Our nimbus env uses the assigned IP as the name.

Reply:
I am aware. I was just making it consistent with the Kube naming as well as to distinguish this from `hostName`. That's why I said it's the observed name or IP address, same as the previous comment. I

Reply:
I was referring to the field description `...IP address of...` which isn't quite right -we don't have fallback code or anything that fetches the ESX host IP.

Reply:
I just copied the original comment. Want me to remove the IP address from the comment then?